tests/qchem/test_factorization.py

```diff
@@ -167,3 +167,37 @@ def test_empty_error(two_tensor):
 
     with pytest.raises(ValueError, match="All eigenvectors are discarded."):
         qml.qchem.factorize(two_tensor, 1e-5, 1e1)
+
+
+@pytest.mark.parametrize(
+    ("one_matrix", "two_tensor", "tol_factor", "coeffs_ref"),
+    [
```

Comment:
Are these obtained manually or via `openfermion`?

Reply:
The are obtained with the DHF solver, added the code.